lib/v2/pkuszcs/xwzx.js

```diff
@@ -0,0 +1,84 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const date = require('@/utils/date');
+
+const map = new Map([
+	[0, {title: '新闻中心', link: 'https://www.ece.pku.edu.cn/xwzx.htm'}],
+	[1, {title: '学院新闻', link: 'https://www.ece.pku.edu.cn/xwzx/xyxw.htm'}],
+	[2, {title: '通知公告', link: 'https://www.ece.pku.edu.cn/xwzx/tzgg.htm'}]
+]);
+
+module.exports = async (ctx) => {
+	const type = Number.parseInt(ctx.params.type);
+
+	if (type !== 0) {
+		const id = map.get(type).link;
+		// 发起 HTTP GET 请求
+		const response = await got({
+	    	method: 'get',
+	    	url: String(id),
+		});
```

Comment:
got request can be simplified
```suggestion
		const response = await got(String(id));
```